test/e2e/app_management_test.go

```diff
@@ -2058,32 +2058,6 @@ definitions:
 		})
 }
 
-func TestAppLogs(t *testing.T) {
```

Comment:
Could it be worth adding a comment or todo to say that these tests should be re-implemented as unit tests?

Reply:
If we have plans to add tests for this, we should maybe create an issue to track it and add a TODO as well.. Else, we would forget it completely.

Reply:
Done! https://github.com/argoproj/argo-cd/issues/13834